src/QMCDrivers/tests/ValidQMCInputSections.h

```diff
@@ -103,13 +103,14 @@ constexpr std::array<const char*, 2> valid_dmc_input_sections{
 constexpr int valid_dmc_input_dmc_index       = 0;
 constexpr int valid_dmc_input_dmc_batch_index = 1;
 
+/** As far as I can tell these are no longer valid */
```

Comment:
I need to update the WFOptDriverInput class, and I'll change these tests then as well.